packages/morph-attr/lib/main.js

```diff
@@ -110,7 +110,7 @@ AttrMorph.prototype.setContent = function (value) {
   if (this.lastValue === value) { return; }
   this.lastValue = value;
 
-  if (this.escaped) {
+  if (this.escaped && this.valueIdentity(value)) {
```

Comment:
Maybe I don't understand what is going on here but if should escape e.g. `this.escape === true`, then the value we are trying to escape should be truthy. Currently we run the block with `undefined` or strings. I added the function instead of doing

```
if (this.escaped && value) {

}
```